client/public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css">
```

Comment:
For consistency with the other packages we have in this project we should add Font Awesome to our package.json as a dependency. Check the docs here for add it using `yarn` commands (we'll only need this in the client package.json since it's used on the front end)
https://fontawesome.com/how-to-use/on-the-web/using-with/react